### PR TITLE
Make `Font` an associated type of `text_input::Renderer`

### PR DESCRIPTION
**Pull request by Imberflur**

### CODE REVIEW

**Comment by Imberflur** [comment]
Updated to resolve conflicts

File: graphics/src/widget/text_input.rs [hunk 1]
```diff
@@ -27,6 +27,8 @@ impl<B> text_input::Renderer for Renderer<B>
 where
     B: Backend + backend::Text,
 {
+    type Font = Font;
+
```
Comment (hecrj) [inline graphics/src/widget/text_input.rs:31]: For consistency, we should remove this blank line between associated types.

File: native/src/renderer/null.rs [hunk 2]
```diff
@@ -103,6 +103,8 @@ impl scrollable::Renderer for Null {
 }
 
 impl text_input::Renderer for Null {
+    type Font = Font;
+
```
Comment (hecrj) [inline native/src/renderer/null.rs:107]: Same here!

**Comment by Imberflur** [comment]
@hecrj Thanks for the review. I removed the blank lines :slightly_smiling_face:

**Pull request merged**